bridges/otelzap/core_test.go

```diff
@@ -40,6 +40,22 @@ func TestCore(t *testing.T) {
 		assert.Equal(t, testValue, value2Result(kv.Value))
 		return true
 	})
+
+	rec.Reset()
+
+	// test child logger with accumulated fields
+	childlogger := logger.With(zap.String("workplace", "otel"))
+	childlogger.Info(testMessage)
```

Comment:
Can we additionally pass an attribute to make sure that all attributes are persevered?